tests/testthat/test-log_shiny_input_changes.R

```diff
@@ -0,0 +1,119 @@
+testthat::describe("log_shiny_input_changes", {
+  create_mock_session <- function() {
```

Comment:
Why to `create_mock_session` manually while there is `shiny:: MockShinySession` or `shiny:: withReactiveDomain`?